geometry_msgs/PoseStamped pose_stamped
string pose_name
---
bool success
